### DISCUSSION TITLE
Skiovox for ChromeOS 120

### DISCUSSION THREAD

**Discussion by synapt0** [3kh0/ext-remover#900] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Note: this is not an issue, just a fix. (lol)

Made fully by @Fighterhen63, I do not hold any respect for this..
Here are the steps..

If you find that u don't have sign in as existing user button when u are on step 7, do this: When you are at the normal sign in screen, turn off wifi, see if u have sign in as existing user there, if u do, click it, then continue to do the method:

1. Sign out of your chromebook.
2. Turn off your wifi

3. Click one of the apps

4. As you click one of the apps, press alt+shift+s as fast as possible

5. This should open the tool bar, wait until the No wifi screen show up.

6. Click the screen brightness settings in the tool bar and click settings(the cog) nothing should happen(tool bar will be closed) and thats normal.

7. Open chrome vox(ctrl+alt+z) then click resources (Like open the menu which has the links like the Search TO method from the old methods), then click on one of the links. Now close chrome vox (same keybinds).

8. Click sign in as existing user put in your school account code and password. (Normal school login.)
If you did it successfully, you should be signed in to school account in the kiosk app and you should see something saying "muti user sign in disabled on this chromebook" just press the ESC button.

9. You should see your school account browser there just close it, you should also see a setting menu somewhere, just try to turn on wifi then close that window. The last window is the fully unblocked one. You can also open incognito, DO NOT because it is still monitored to your school account so don't use that. Something you need to do it click the three dots and open "New Window" this will allow windows to be opened so you can install skiviox. This should be your primary window.

If anyone has any questions feel free to reach out to Fighterhen63!

Originally posted by @Fighterhen63 in https://github.com/bypassiwastaken/skiovox-helper/issues/90#issuecomment-1877083009

Notes from @funkymonkeyo: If the settings tab pop up doesn’t show up, navigate to chrome://network/#select on the unblocked window.
Edit: Readability.

**Comment by CGRmags** [comment]
@funkymonkeyo I cant do any off this I can do up to Chrome vox thing but when I press the link nothing shows up and im lost

**Reply by synapt0** [reply]
It shouldn’t pop up with anything yet, just continue through to the next step.

**Comment by woodpecker4w00d** [comment]
I don't have the sign in as existing user thing

**Reply by synapt0** [reply]
Then sign out and choose a different app.

**Comment by xd102401** [comment]
where are the resources on chromevox? because when i close settings its just black

**Reply by synapt0** [reply]
What tabs did you close, or were there any?

**Comment by CJBrandi** [comment]
@funkymonkeyo I can't click on the brightness settings, they don't work😕![image](https://github.com/3kh0/ext-remover/assets/102370270/d6c30417-ff51-46c9-9c4b-fdb056f8d334)

**Reply by nathblade16** [reply]
your kiosk app is likely incompatible with skiovox or your doing it too early, your meant to do it on the no wifi screen

**Comment by 4kblvntz** [comment]
what do you mean the resources cause i don’t see it. Could you please make a video on how to do it, please and thank you.

**Reply by nathblade16** [reply]
you need to press search + o + t after chromevox
